docs/howto/team-open-science-mindset.md

```diff
@@ -21,46 +21,77 @@ We want our community ontology editors and developers to be _excited_ to be part
 
 1. **Take ownership of our community ontologies**
 
-- Ensure that you see your issues and Pull Requests through to the end
-- Feel empowered to nag reviewers or experts to help
+    - Ensure that you see your issues and Pull Requests through to the end
```

Comment:
Minor edit suggestion:
"... Issues and Pull Requests ..."
or 
"... issues and pull requests ..."